VRDR/DeathRecord_util.cs

```diff
@@ -1075,6 +1075,56 @@ public static void updateGivenHumanName(string[] value, List<HumanName> names)
                 names.Add(name);
             }
         }
+
+        /// <summary>Helper method to validate that all PartialDate and PartialDateTime exensions are valid and have the valid required sub-extensions.</summary>
+        /// <param name="bundle">The bundle in which to validate the PartialDate/Time extensions.</param>
+        private static void ValidatePartialDates(Bundle bundle)
+        {
+            System.Text.StringBuilder errors = new System.Text.StringBuilder();
+            List<Resource> resources = bundle.Entry.Select(entry => entry.Resource).ToList();
+
+            foreach (Hl7.Fhir.Model.Resource resource in resources)
+            {
+                foreach (DataType child in resource.Children.Where(child => child.GetType().IsSubclassOf(typeof(DataType))))
```

Comment:
I don't love the use of `.IsSubclassOf` here. All it needs is to be filtered and cast to the highest-level type that supports Extensions, is there a better way to do this that the FHIR library supports?

Reply:
Not sure that there's a better way to do this.